addons/hr_expense/migrations/11.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,32 @@
+---Fields in module 'hr_expense'---
+hr_expense   / account.move.line        / expense_id (many2one)         : NEW relation: hr.expense
```

Comment:
Is there a way to link old expenses to created move lines. I think this is important to be researched.